### ISSUE TITLE
themekit.AssetError on theme watch command

### ISSUE THREAD

**Issue by stevebosworth** [Shopify/themekit#100]
Found an issue when running the `theme watch` command. 

Returns a 401 error with the message: `json: cannot unmarshal string into Go value of type themekit.AssetError` when you save a liquid file

**Comment by jterskine** [comment]
+1

**Comment by csaunders** [comment]
If you have more information you could provide into what is happening that would be great. +1's don't provide more debugging information and as a result just generate noise.

A tool that is vital for providing insight into what is going on is [mitmproxy](http://mitmproxy.org/doc/mitmproxy.html). Themekit supports proxies by simply setting the `proxy` field in a configuration.

Look at the API response from Shopify and include what was sent (minus sensitive data) and include it your bug report.

**Comment by heymynameisrob** [comment]
Also getting a similar error. 
```

[500]Could not perform Update to assets/style.scss.liquid at foobar.myshopify.com
	json: cannot unmarshal string into Go value of type themekit.AssetError
```

**Comment by frehner** [comment]
Just starting receiving this error suddenly (after several months of development without any issues or errors). The only thing that changed was that I upgraded to 0.4.0 from 0.3.6, and suddenly was getting the error in both my watch and replace commands. 

I tried to downgrade to 0.3.6 to see if it would fix it, but 0.3.6 now suddenly has the error as well.

OS: OSX.  

I'll try to get into mitmproxy and see if I can find what's going on.
